### ISSUE TITLE
Light mode / theme

### ISSUE THREAD

**Issue by psidex** [DioxusLabs/docsite#408]
I struggle to read white text on a black background, it would massively help accessibility if there was an optional light mode / theme

**Comment by DogeDark** [comment]
We have one but it is tied to the system/browser theme. We should add an override switch for it.
